[FR] Add `Help_centre\Tournament_bans` (#7200)
ppy/osu-wiki: wiki/Help_centre/Tournament_bans/fr.md

# Bannissement des tournois

*Page principale : [Centre d'aide](/wiki/Help_centre)*

> Un bannissement des tournois empêche un joueur de s'impliquer dans les tournois officiels, à tous les niveaux. Cela inclus le jeu et la modération/gestion, ou tout autres types d'assistances.

Reviewer: ```suggestion
Un bannissement des tournois empêche un joueur de s'impliquer dans les tournois officiels, à tous les niveaux. Cela inclut le jeu et la modération/gestion, ou tout autre type d'assistance.
```

> Un bannissement des tournois est très rarement prononcées en tant que telles, mais en général, les joueurs qui retournent dans un appel pour une [restriction de compte](/wiki/Help_centre/Account_restrictions) sont soumis a une durée d'au moins 1 an.

Reviewer 2: ```suggestion
Un bannissement des tournois est très rarement prononcé en tant que tel, mais en général, les joueurs qui retournent dans un appel pour une [restriction de compte](/wiki/Help_centre/Account_restrictions) sont soumis a une durée d'au moins 1 an.
```

> En raison de l'importance des enjeux d'un tournoi, les bannissements des tournois sont traiter avec le plus grand sérieux, il n'y a donc pas d'indulgence comme pour les restriction de compte.

Reviewer 2: ```suggestion
En raison de l'importance des enjeux d'un tournoi, les bannissements des tournois sont traités avec le plus grand sérieux, il n'y a donc pas d'indulgence comme pour les restriction de compte.
```

## Quel sont les causes d'un bannissement temporaire d'un tournoi ?

> Toute violation sévère des [règles de la communauté](/wiki/Rules) qui ont lieu lors d'un [tournois officiellement supporté](/wiki/Tournaments/Official_support) est souvent la cause d'un bannissement d'un tournoi, même si cela n'aurait eu pour conséquence une infraction en dehors du cadre de la compétition.

Reviewer 2: ```suggestion
Toute violation sévère des [règles de la communauté](/wiki/Rules) qui a lieu lors d'un [tournoi officiellement supporté](/wiki/Tournaments/Official_support) est souvent la cause d'un bannissement d'un tournoi, même si cela n'aurait eu pour conséquence une infraction en dehors du cadre de la compétition.
```

Comme exemple, un tel comportement inclurait le dessin ou l'écriture de symboles haineux/insultant (croix gammée par exemple) en utilisant le cursor smoke dans les matchs diffusé.

> Tous les utilisateurs retournant dans le jeu après une procédure de restriction dans le cadre des conditions d'appel standard sont également soumis à un [bannissement temporaire des tournois d'une duré de 1 an](/wiki/Help_centre/Account_restrictions#common-restriction-reasons-and-cooldowns), qui peut être prolongé à la discrétion de [l'Équipe de support aux comptes](/wiki/People/The_Team/Account_support_team).

Reviewer 2: ```suggestion
Tous les utilisateurs retournant dans le jeu après une procédure de restriction dans le cadre des conditions d'appel standard sont également soumis à un [bannissement temporaire des tournois d'une duré de 1 an](/wiki/Help_centre/Account_restrictions#common-restriction-reasons-and-cooldowns), qui peut-être prolongé à la discrétion de [l'Équipe de support aux comptes](/wiki/People/The_Team/Account_support_team).
```

## Quel sont les causes d'un bannissement permanent d'un tournoi ?

> Toute utilisation d'un logiciel tiers de triche ou toute autres méthodes pour gagner un avantages déloyal vis a vis des autres participants dans un tournois officiellement supporté résultera en un bannissement permanent. Dans certaines situations, plus particulièrement dans les cas d'abus excessif peuvent conduire a une telle sanction.

Reviewer 2: ```suggestion
Toute utilisation d'un logiciel tiers de triche ou toute autres méthodes pour gagner un avantages déloyal vis a vis des autres participants dans un tournoi officiellement supporté résultera en un bannissement permanent. Dans certaines situations, des cas d'abus particulièrement excessifs peuvent également donner lieu à une telle sanction.
```

Cela inclus des situations comme :

- Comme évoqué, toute utilisation d'un logiciel tiers de triche ou d'un utilitaire pour gagner un avantage
- L'utilisation de plusieurs comptes pour entrer dans un tournoi
- La tentative de contourner un silence ou toute autre sanction en utilisant un autre compte
- Le partage de compte lors d'un tournois officiellement supporté
- La conspiration pour obtenir ou utiliser des informations privés pour gagner ou obtenir un avantage (par exemple, accéder au mappool, obtenir des informations des autres équipes, etc...)
- D'abuser de la confiance d'autrui pour gagner ou obtenir un avantage (par exemple, manipuler le déroulement du tournoi, de défavoriser les équipes en changeant leurs compositions, ou disqualifier les participants sans raisons valables ou raisonnement)

> - Organiser des campagnes ciblé d'insulte envers les participants ou des organisateurs bien en dehors de la limite de la critique

Reviewer 2: ```suggestion
- Organiser des campagnes ciblées d'insulte envers les participants ou des organisateurs bien en dehors de la limite de la critique
```

## Est t'il possible de faire appel d'un bannissement des tournois ?

Qu'il soit temporaire ou permanent, **il n'est pas possible de faire appel d'un bannissement des tournois**.

> Dans des cas particuliers, [l'Équipe de support aux comptes](/wiki/People/The_Team/Account_support_team) peut étudier des cas individuel dans leurs discrétion et retirer ou réappliquer des sanctions appropriée pour être sûr que cela reste cohérent avec les situations passés.

Reviewer: ```suggestion
Dans des cas particuliers, [l'Équipe de support aux comptes](/wiki/People/The_Team/Account_support_team) peut étudier des cas individuels à leur discrétion et retirer ou réappliquer des sanctions appropriées pour être sûr que cela reste cohérent avec les situations passées.
```

> Dans des cas particuliers, [l'Équipe de support aux comptes](/wiki/People/The_Team/Account_support_team) peut étudier des cas individuel dans leurs discrétion et retirer ou réappliquer des sanctions appropriée pour être sur que cela reste cohérent avec les situations passés.

Reviewer 2: ```suggestion
Dans des cas particuliers, [l'Équipe de support aux comptes](/wiki/People/The_Team/Account_support_team) peut étudier des cas individuel dans leurs discrétion et retirer ou réappliquer des sanctions appropriée pour être sûr que cela reste cohérent avec les situations passés.
```

> ## Qu'est t'il possible de faire en cas de bannissement d'un tournoi ?

Reviewer: ```suggestion
## Que puis-je faire en cas de bannissement des tournois ?
```

> Vous pouvez continuer à jouer dans les tournois communautaire qui ne requiert pas de support officiel ou requiert une vérification à la discrétion des personnes qui organisent ce genre d'événements.

Reviewer: ```suggestion
Vous pouvez continuer à jouer dans les tournois communautaires qui n'ont pas de support officiel ou de screening à la discrétion des organisateurs de ce genre d'événements.
```

> Les informations concernant votre status de bannissement des tournois peut être communiqué aux organisateurs des tournois avant de postuler, qu'il soit officiel ou non

Reviewer: ```suggestion
Les informations concernant votre statut de bannissement des tournois peuvent être communiquées aux organisateurs des tournois sur demande, que le tournoi soit officiel ou non.
```

> Les informations concernant votre status de bannissement des tournois peut être communiquer aux organisateurs des tournois avant de postuler, qu'il soit officiel ou non

Reviewer 2: ```suggestion
Les informations concernant votre status de bannissement des tournois peut être communiqué aux organisateurs des tournois avant de postuler, qu'il soit officiel ou non
```

> ## Raisons communes de bannissement d'un tournoi et durée

Reviewer: ```suggestion
## Raisons communes de bannissement des tournois et durées
```

> | Raisons du bannissement du tournoi | Durée | Commentaire |

Reviewer: ```suggestion
| Raison du bannissement des tournois | Durée | Commentaire |
```

| :-- | :-- | :-- |

> | Une mauvaise conduite d'un joueur lors d'un tournois officiellement supporté | 1 ou 2 an(s) |  |

Reviewer: ```suggestion
| Mauvaise conduite d'un joueur lors d'un tournoi officiellement supporté | 1 ou 2 ans |  |
```

> | Un mauvaise conduite d'un joueur lors d'un tournois officiellement supporté | 1 ou 2 an(s) |  |

Reviewer 2: ```suggestion
| Une mauvaise conduite d'un joueur lors d'un tournois officiellement supporté | 1 ou 2 an(s) |  |
```

> | Un retour dans les conditions standard d'appels | 1 ou 2 an(s) | Peut-être prolongé a la discrétion de l'Équipe de support aux comptes |

Reviewer: ```suggestion
| Retour dans les conditions standards d'appel | 1 ou 2 ans | Peut-être prolongé à la discrétion de l'Équipe de support aux comptes |
```

> | Un retour dans les conditions standard d'appels | 1 ou 2 an(s) | Peut être prolongé a la discrétion de l'Équipe de support aux comptes |

Reviewer 2: ```suggestion
| Un retour dans les conditions standard d'appels | 1 ou 2 an(s) | Peut-être prolongé a la discrétion de l'Équipe de support aux comptes |
```

> | Une utilisation de plusieurs comptes ou partage de compte lors d'un tournois officiellement supporté | Permanent | Engendre également une restriction |

Reviewer: ```suggestion
| Utilisation de plusieurs comptes ou partage de compte lors d'un tournoi officiellement supporté | Permanent | Engendre également une restriction |
```

> | Une utilisation d'un logiciel tiers ou de tout utilitaire de triche durant les tournois officiellement supporté | Permanent | Engendre également une restriction |

Reviewer: ```suggestion
| Utilisation d'un logiciel tiers ou de tout utilitaire de triche lors d'un tournoi officiellement supporté | Permanent | Engendre également une restriction |
```

> | Une utilisation d'informations de tournois privé/confidentiel pour gagner ou conférer un avantage déloyal | Permanent | Peut causer également une restriction |

Reviewer: ```suggestion
| Utilisation d'informations de tournois privées/confidentielles pour gagner ou conférer un avantage déloyal | Permanent | Peut également causer une restriction |
```

> | Un abus sévère ou harcèlement vis a vis d'un organisateur/personnel/participants du tournois | Permanent | Peut causer également une restriction |

Reviewer: ```suggestion
| Abus sévère ou harcèlement envers des organisateurs/staff/participants du tournoi | Permanent | Peut également causer une restriction |
```
